src/main/java/nextstep/ladder/player/Players.java

```diff
@@ -28,10 +28,11 @@ public static Players of(String names) {
                 .collect(Collectors.toList());
     }
 
-    public Player findByName(String name) {
+    public int findByName(String name) {
```

Comment:
```suggestion
    public int findPositionByName(String name) {
```

메서드의 기능이 변경되며 네이밍도 같이 변경해주면 좋을 것 같아요 :)